rancher-ui-driver-linode.yaml

```
@@ -0,0 +1,52 @@
+package:
+  name: rancher-ui-driver-linode
+  version: "0.7.0"
+  epoch: 0
+  description: UI for Linode options when used as a Rancher Node Template
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - bash
+      - busybox
+      - nodejs
+      - npm
+      - nvm
+      - yarn
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/linode/rancher-ui-driver-linode
+      tag: v${{package.version}}
+      expected-commit: b5a9fd67a069081a7c66f52b0d7870b4f9748661
+
+  - runs: |
+      npm install
+      npm run build
+
+  - name: Install files
+    runs: |
+      mkdir -p ${{targets.destdir}}/usr/share/rancher/ui/assets/rancher-ui-driver-linode
+      cp -r assets/* ${{targets.destdir}}/usr/share/rancher/ui/assets/rancher-ui-driver-linode
+      cp -r dist/* ${{targets.destdir}}/usr/share/rancher/ui/assets/rancher-ui-driver-linode
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: linode/rancher-ui-driver-linode
+    strip-prefix: v
+    tag-filter: v
+    use-tag: true
+
+test:
+  pipeline:
+    - name: Ensure files existance
+      runs: |
+        for f in linode.svg component.js component.css linode.svg; do
```

```suggestion
        for f in component.js component.css linode.svg; do
```
nit: we can consider only checking it once.